scripts/data_generation_script.py

```diff
@@ -6,59 +6,66 @@
 import sys
 import os
 
-sys.path.insert(1, os.path.join(sys.path[0], '..'))
+sys.path.insert(1, os.path.join(sys.path[0], ".."))
 from config import *
 
+
 def none_or_str(value):
-    print('none_or_str')
+    print("none_or_str")
     print(value)
     print(type(value))
-    if value == 'None':
+    if value == "None":
         return None
     return value
 
+
 def none_or_int(value):
-    print('none_or_int')
+    print("none_or_int")
     print(value)
     print(type(value))
-    #print('printing type of value supplied to non_or_int ', type(int(value)))
-    if value == 'None':
+    # print('printing type of value supplied to non_or_int ', type(int(value)))
+    if value == "None":
         return None
     return int(value)
 
-if __name__ == "__main__":
 
+if __name__ == "__main__":
     # Interface ----
     CLI = argparse.ArgumentParser()
-    CLI.add_argument("--config_path",
-                     type = none_or_str,
-                     default = None)
-    CLI.add_argument('--data_gen_base_path',
-                     type = none_or_str,
-                     default = None)
-    
+    CLI.add_argument("--config-path", type=none_or_str, default=None)
+    CLI.add_argument("--data-gen-base-path", type=none_or_str, default=None)
+
     args = CLI.parse_args()
     print(args)
 
-    config_dict = get_data_generator_config(yaml_config_path = args.config_path,
-                                            base_path = args.data_gen_base_path)['config_dict']
+    config_dict = get_data_generator_config(
+        yaml_config_path=args.config_path, base_path=args.data_gen_base_path
+    )["config_dict"]
+
+    assert args.config_path is not None, (
+        "You need to supply a config file path to the script"
+    )
+
+    print("Printing config specs: ")
```

Comment:
maybe we want to exchange print for log everywhere?
(could be separate issue ofc)